en/flight_modes/README.md

```diff
@@ -1,15 +1,415 @@
+</tr>
+
+<tr id="followme_mc">
+ <td><a href="../flight_modes/follow_me.md">Follow Me</a></td>
+ <td colspan="3">Auto</td>
+ <td><a href="#key_position_fixed"><img src="../../assets/site/position_fixed.svg" title="Position fix required (e.g. GPS)" width="20px" /></a></td>
+ <td>Vehicle autonomously follows a user using an Android phone/tablet running QGC.</td>
+</tr>
+
+<tr id="offboard_mc">
+ <td><a href="../flight_modes/offboard.md">Offboard</a></td>
+ <td colspan="3">Auto</td>
+ <td><a href="#key_position_fixed"><img src="../../assets/site/position_fixed.svg" title="Position fix required (e.g. GPS)" width="20px" /></a></td>
+ <td>Vehicle obeys a position, velocity or attitude setpoint provided over MAVLink (often from a companion computer connected via serial cable or wifi).</td>
+</tr>
+ 
+</tbody></table>
+
+
+## VTOL
+
+VTOL vehicles support both fixed-wing and multicopter flight modes, executing them based on the current vehicle mode (MC or FW).
+
+VTOL supports <a href="../flight_modes/offboard.md">Offboard</a> mode in either configuration.
+
+
+## Key
+
+Key for understanding the table is as follows:
+
+Symbol | Description
+--- | ---
+M | Manual control via RC sticks. RC input is sent directly to the output mixer.
+S | Assistance from autopilot to stabilize the attitude. RC input is required. Position of RC stick maps to the orientation of vehicle.
+S<sub>rate</sub> |  Assistance from autopilot to stabilize the attitude rate. RC input is required. Position of RC stick maps to the rate of rotation of vehicle in that orientation.
+S<sup>+</sup> | Assistance from autopilot to hold position or altitude against wind. RC input is required.
+Auto | This mode is automatic (RC control is disabled by default except to change modes).
+<span id="key_position_fixed"></span><img src="../../assets/site/position_fixed.svg" title="Position fix required (e.g. GPS)" width="20px" /> | Sensor(s) that measures position/height needed e.g. optical flow, GPS+barometer, visual-inertial odometry. <p id="baro_only"><b>++</b> - <em>Altitude Mode</em> only requires altitude sensor (e.g. Baro, Rangefinder).</p>
+<span id="key_difficulty">[<img src="../../assets/site/difficulty_1.svg" title="Difficulty (Easiest)" width="20px" />&nbsp;<img src="../../assets/site/difficulty_2.svg" title="Difficulty (Easy)" width="20px" />&nbsp;<img src="../../assets/site/difficulty_3.svg" title="Difficulty (Medium)" width="20px" />&nbsp;<img src="../../assets/site/difficulty_4.svg" title="Difficulty (Medium-hard)" width="20px" />&nbsp;<img src="../../assets/site/difficulty_5.svg" title="Difficulty (Hard)" width="20px" />&nbsp;<img src="../../assets/site/difficulty_6.svg" title="Difficulty (Hardest)" width="20px" />](#key_difficulty) | Flight mode difficulty (easy to hard).
+
+
+Abbreviations:
+  * RPY: Roll, Pitch, Yaw
+  * RPT: Roll, Pitch Throttle
+
+
+
+<!-- 

```

Comment:
Note, I just commented out the duplicated material. This will be deleted when I'm sure *everything* has been captured in the other docs. Won't be long.